### PR TITLE
Optimizations for ReadySetCyber Header, Dashboard, and Assessment

### LINKED ISSUE #153: Develop Custom RSC Header
Need assets from RSC so we can match their design.

### PR DESCRIPTION
**Pull request by hawkishpolicy**
#  #
- Updated RSC Header component
- Updated RSC Dashboard component
- Updated RSC Detail component
- Updated RSC Question component
## 💭 Motivation and context ##
 Needed responsive design for Header, Dashboard, and Assessment components.

closes #153  

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All new and existing tests pass.

## ✅ Pre-merge checklist ##

- [ ] Revert dependencies to default branches.
- [ ] Finalize version.

## ✅ Post-merge checklist ##

- [ ] Create a release.

### CODE REVIEW

**Review by cduhn17 — APPROVED** [review]
LGTM

File: frontend/src/components/ReadySetCyber/Assets/ReadySetCyberLogo.png
Comment (rapidray12) [inline frontend/src/components/ReadySetCyber/Assets/ReadySetCyberLogo.png:1]: I like it.  Very stylish.  We need something like this for Crossfeed.  The Crossfeed lighthouse has to go.

**Comment by cduhn17** [comment]
@hawkishpolicy , 
Did the logo go threw an approval process?

**Comment by hawkishpolicy** [comment]
It's what is on the Figma

**Comment by hawkishpolicy** [comment]
I was instructed to match the Figma.

**Review by rapidray12 — COMMENTED** [review]
I have some issues with client side processing but otherwise it looks good to me

**Review by rapidray12 — APPROVED** [review]

**Review by rapidray12 — APPROVED** [review]

**Review by Matthew-Grayson — CHANGES_REQUESTED** [review]
There is a failure in the frontend pipeline / test check by GitHub Actions.

**Review by Matthew-Grayson — APPROVED** [review]

**Comment by Matthew-Grayson** [comment]
Looks great!

**Pull request merged**